imports/plugins/core/payments/server/no-meteor/mutations/enablePaymentMethodForShop.js

```diff
@@ -0,0 +1,36 @@
+import ReactionError from "@reactioncommerce/reaction-error";
+import SimpleSchema from "simpl-schema";
+import { Shops } from "/lib/collections";
```

Comment:
In no-meteor code we use the collections off `context` rather than importing them. The ones in /lib/collections are Meteor collections while the ones on context are straight from the `mongodb` NPM package.

```js
const { Shops } = context.collections;
```

docs ref: https://docs.reactioncommerce.com/docs/graphql-developing#using-mongodb-in-graphql